Error while parsing `message_fr.xml`

Problem
--
An error occured while reading file `message_fr.xml`.

Version
--
SpotBugs 4.8.0

Error Log
--
```java
edu.umd.cs.findbugs.PluginException: Couldn't parse "jar:file:/home/gradle/.gradle/caches/modules-2/files-2.1/com.github.spotbugs/spotbugs/4.8.0/b4f9cd4fc6352be6226efa0c5f59f0dc10219732/spotbugs-4.8.0.jar!/messages_fr.xml"
	at edu.umd.cs.findbugs.PluginLoader.addCollection(PluginLoader.java:1336)
	at edu.umd.cs.findbugs.PluginLoader.getMessageDocuments(PluginLoader.java:1235)
	at edu.umd.cs.findbugs.PluginLoader.loadPluginComponents(PluginLoader.java:750)
	at edu.umd.cs.findbugs.PluginLoader.<init>(PluginLoader.java:229)
	at edu.umd.cs.findbugs.PluginLoader.getPluginLoader(PluginLoader.java:1378)
	at edu.umd.cs.findbugs.Plugin.addCustomPlugin(Plugin.java:671)
	at edu.umd.cs.findbugs.Plugin.addCustomPlugin(Plugin.java:663)
	at edu.umd.cs.findbugs.Plugin.loadCustomPlugin(Plugin.java:655)
	at edu.umd.cs.findbugs.Plugin.loadCustomPlugin(Plugin.java:648)
	at edu.umd.cs.findbugs.FindBugsCommandLine.handleOptionWithArgument(FindBugsCommandLine.java:147)
	at edu.umd.cs.findbugs.TextUICommandLine.handleOptionWithArgument(TextUICommandLine.java:619)
	at edu.umd.cs.findbugs.config.CommandLine.parse(CommandLine.java:333)
	at edu.umd.cs.findbugs.config.CommandLine.parse(CommandLine.java:300)
	at edu.umd.cs.findbugs.FindBugs.processCommandLine(FindBugs.java:345)
	at edu.umd.cs.findbugs.FindBugs2.main(FindBugs2.java:1221)
Caused by: org.dom4j.DocumentException: Error on line 1723 of document  : Le type d'élément "MessageCollection" doit se terminer par la balise de fin correspondante "</MessageCollection>".
	at org.dom4j.io.SAXReader.read(SAXReader.java:511)
	at org.dom4j.io.SAXReader.read(SAXReader.java:408)
	at edu.umd.cs.findbugs.PluginLoader.addCollection(PluginLoader.java:1333)
	... 14 more
Caused by: org.xml.sax.SAXParseException; lineNumber: 1723; columnNumber: 4; Le type d'élément "MessageCollection" doit se terminer par la balise de fin correspondante "</MessageCollection>".
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:178)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1465)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(XMLDocumentFragmentScannerImpl.java)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2899)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:542)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:889)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:825)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1224)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:637)
	at org.dom4j.io.SAXReader.read(SAXReader.java:494)
	... 16 more
```

--- a/spotbugs/etc/messages_fr.xml
+++ b/spotbugs/etc/messages_fr.xml
@@ -1720,7 +1720,6 @@ Cependant l'initialisation du champ comporte plusieurs écriture, ce qui requier
 ]]>
   </Details>
  </BugPattern>
- </BugPattern>
  <BugPattern type="MS_SHOULD_BE_FINAL">
   <ShortDescription>Un champ n'est pas final alors qu'il devrait l'être</ShortDescription>
   <LongDescription>{1} n'est pas final mais devrait l'être</LongDescription>

